resources/pdns_authoritative_service_debian_sysvinit.rb

```diff
@@ -1,96 +0,0 @@
-#
```

Comment:
One only wants to create a new pdns instance with one's configuration, the implementation is not that important. The `pdns` package guaranty their `init` scripts will work all the time no matter the distribution; so relying on that hypotheses, I though maintaining `init` scripts for different distributions seems overkill (if pdns maintainers change the way the binary is launch you will have to release a new cookbook to handle that). On top of that, the resource allows anyone to add their own [init scripts](https://github.com/dnsimple/chef-pdns/pull/66/files#diff-cb4116089cbc21319bcd68fbb4e74125R30).

Don't you think it is easier to maintain?